index.cjs

```diff
@@ -165,6 +165,24 @@ module.exports = {
     // Warn on restricted syntax
     'no-restricted-syntax': [
       'warn',
+      {
+        selector:
+          "ExpressionStatement CallExpression[callee.object.name='document'][callee.property.name='querySelector']",
+        message: `Using document.querySelectorAll() can give unexpected results specially if there are multiple elements that match the query (for example, when the component is used more than once), as an advise use State or Refs instead
+        https://github.com/reactjs/reactjs.org/issues/4626#issuecomment-1117535930`,
```

Comment:
maybe the lines after the newlines should be at the beginning of the line? eg. with no indentation?